lib/models/movie.js

```diff
@@ -7,7 +7,7 @@ module.exports = Bookshelf.Model.extend({
   serialize: function () {
     return {
       id: this.get('id'),
-      title: this.get('title'),
+      title: this.get('name'),
```

Comment:
we cant start using this new column just yet. if i do a GET on an existing movie, the `title` property will return back `null` since the `name` column wont have been backfilled yet.